dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -560,7 +562,35 @@ ANNO_TARGETS_JANDEX_DUPLICATE_CLASS.useraction=CMSG0002
 
+# # {0} A string: An application name.
+# # {1} A string: A module name.
+# # {2} A number: The total count of module locations.
+# # {3} A number: The count of module locations which had Jandex indexes.
+# # {4} A number: The total count of module classes.
+# # {5} A number: The count of module classes which were read from a Jandex index.
+ANNOCACHE_JANDEX_USAGE=CWWKC0093I: Jandex coverage of module {1} in application {0}: Read Jandex indexes for {3} out of {2} module locations; Jandex indexes provided {5} out of {4} module classes.
+ANNOCACHE_JANDEX_USAGE.explanation=This shows the proportion of a module which was scanned for annotations using Jandex indexes.  Module locations are either the module itself (for example, for an EJB Jar file), or are the WEB-INF/classes folder and WEB-INF/lib Jar files (for a WAR file).
+ANNOCACHE_JANDEX_USAGE.useraction=If Jandex indexes are intended to be read for the application then there is no action.  If Jandex indexes are not to be read for the application, then set the useJandex="false" attribute in the application element in server.xml. Alternatively, set useJandex="false" in the applicationManger element in server.xml to ignore Jandex indexes in all applications.
+
+# # {0} A string: Message text from an exception.
+ANNO_TARGETS_CACHE_EXCEPTION=CWWKC0101W: Annotation processing cache error: {0}
+ANNO_TARGETS_CACHE_EXCEPTION.explanation=An exception occured while accessing annotations cache data.
+ANNO_TARGETS_CACHE_EXCEPTION.useraction=See the exception and the associated FFDC message for moreinformation.
+
+# # {0} A string: A class name.
+# # {1} A string: A location which contains the class: The path to the JAR file which contains the class, or the path to the directory which contains the class, or the name of the internal data structure which contains the class.
+# # {2} A string: Message text from an exception.
+ANNO_TARGETS_CLASS_SCAN_EXCEPTION: CWWKC0103W: An exception occurred while scanning class {0} of {1}.  The exception was {2}.
+ANNO_TARGETS_CLASS_SCAN_EXCEPTION.explanation: An error occurred while scanning class and annotations data.  The associated module may function incorrectly.
+ANNO_TARGETS_CLASS_SCAN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+
+# # {0} A String: An internal print string of the data structure which was used to locate the class.
+# # {1} A String: A class name.
+# # {2} A String: A class name.
+ANNO_TARGETS_CLASS_NAME_MISMATCH=CWWKC0105W: Class scanning internal error: The visitor [ {0} ] read class [ {1] ] from a resource which matches class [ {2} ].
+ANNO_TARGETS_CLASS_NAME_MISMATCH.explanation=The name recorded within the '.class' resource does not match the class name obtained from the class resource. The class was moved outside of the directory which matches the class's qualified name.  The class will be ignored.
```

Comment:
Change
'The name recorded within the '.class' resource does not match the class name obtained from the class resource. The class was moved outside of the directory which matches the class's qualified name.  The class will be ignored.'
to
'The name that is recorded within the '.class' resource does not match the class name that is obtained from the class resource. The class was moved outside of the directory,  which matches the class's qualified name.  The class will be ignored.'

Reply:
Updated message to clarify.